run/shared/setupConfigFiles.sh

```diff
@@ -111,6 +111,8 @@ function set_common_settings() {
     #------------------------------------------------------------------------
     if [[ "x${sim_extra_option}" == "xbenchmark" ]]; then
 
+	sed_ie 's|EFYO|CYS|'                          HEMCO_Config.rc  # Set missing species to background values
```

Comment:
I wonder if this should have an additional criteria to make sure we don't inadvertently put EFY0 elsewhere in HEMCO_Config.rc and then inadvertently change it for benchmark simulations. CYS has other differences from EFY0 besides skipping missing variables, right? Could this potentially cause problems later on, or is EFY0 definitely only going to be used for restart files?

Reply:
@lizziel: right now, EFYO is only used for restart files.

Reply:
Now fixed in commit ddde7ea585.